### PR TITLE
Add missing gettext imports to run_uh.py

### PR DESCRIPTION
**Pull request by Niektory**
Refs: #2516

### CODE REVIEW

File: run_uh.py [hunk 1]
```diff
@@ -176,6 +177,7 @@ def main():
 	try:
 		import yaml
 	except ImportError:
+		from horizons.i18n import gettext as _
```
Comment (MasterofJOKers) [inline run_uh.py]: These lines don't seem to need the import, but rather need a removal of the `_()`, as other languages are only initialized on line 190 of this file in `horizons.main.start()` (which calls `horizons.engine.engine.Fife.init()`, which calls `horizons.engine.settings.Settings.apply()`, which calls `horizons.i18n.change_language()`).

**Comment by Niektory** [comment]
Amended the PR.

**Comment by Niektory** [comment]
Oh, right. That's what I get for only looking at old commits and not current code. Changed.

**Comment by LinuxDonald** [comment]
Thank you very much for your help Niektory :) 👍

**Pull request merged**